src/script/common/c_converter.cpp

```diff
@@ -271,6 +291,16 @@ v3f checkFloatPos(lua_State *L, int index)
 	return check_v3f(L, index) * BS;
 }
 
+float checkFloatYaw(lua_State *L, int index)
+{
+	return check_float(L, index) * core::RADTODEG;
+}
+
+float readFloatYaw(lua_State *L, int index)
+{
+	return read_float(L, index) * core::RADTODEG;
+}
```

Comment:
I can certainly remove the `readFloatYaw` and `checkFloatYaw` helpers. But I do feel there is value in keeping the new `read_float` and `check_float` helpers since they perform important floating-point sanity checks.